### PR TITLE
enhance: add metrics for load segment progress

### PR DESCRIPTION
**Pull request by weiliu1031**
This PR add metrics for load segment progress:
1. add metrics for load segment/index concurrency
2. add metrics for load index latency
3. change load segment latency's time unit to ms

### CODE REVIEW

File: pkg/metrics/querynode_metrics.go [hunk 1]
```diff
@@ -479,6 +479,28 @@ var (
...
+			loadTypeName,
+		})
+
+	QueryNodeLoadIndexLatency = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryNodeRole,
+			Name:      "load_index_latency",
+			Help:      "latency of load per segment's index",
+			Buckets:   []float64{0.1, 0.5, 1, 5, 10, 20, 50, 100, 300, 600, 1200}, // unit seconds
```
Comment (congqixia) [inline pkg/metrics/querynode_metrics.go:500]: Maybe use pre defined bucket 
also the comment here is misleading, the unit shall be milliseconds

File: pkg/metrics/querynode_metrics.go [hunk 2]
```diff
@@ -479,6 +479,28 @@ var (
...
+			loadTypeName,
+		})
+
+	QueryNodeLoadIndexLatency = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryNodeRole,
+			Name:      "load_index_latency",
+			Help:      "latency of load per segment's index, in milliseconds",
+			Buckets:   buckets, // unit milliseconds
```
Comment (LoveEachDay) [inline pkg/metrics/querynode_metrics.go:500]: Should the default bucket be enough for the load index latency? The  largest of value for default buckets is 130 second.

File: internal/querynodev2/segments/segment_loader.go [hunk 3]
```diff
@@ -654,7 +661,7 @@ func (loader *segmentLoader) Load(ctx context.Context,
 		log.Info("load segment done", zap.Int64("segmentID", segmentID))
 		loader.notifyLoadFinish(loadInfo)
 
-		metrics.QueryNodeLoadSegmentLatency.WithLabelValues(fmt.Sprint(paramtable.GetNodeID())).Observe(tr.ElapseSpan().Seconds())
+		metrics.QueryNodeLoadSegmentLatency.WithLabelValues(fmt.Sprint(paramtable.GetNodeID())).Observe(float64(tr.ElapseSpan().Milliseconds()))
 		return nil
```
Comment (LoveEachDay) [inline internal/querynodev2/segments/segment_loader.go:665]: We'd change the bucket for LoadSegment Latency.

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**